tests/scripts/github-action-helper.sh

```diff
@@ -192,6 +192,16 @@ function deploy_manifest_with_local_build() {
   kubectl create -f $1
 }
 
+function replace_ceph_image() {
+  local file="$1"  # parameter 1: the file in which to replace the ceph image
+  local ceph_image="${2:-}"  # parameter 2: the new ceph image to use
+  if [[ -z ${ceph_image} ]]; then
```

Comment:
You don't need double brackets when calling `-z` which is an argument from the `test` command.